hazelcast/src/main/java/com/hazelcast/config/DiscoveryStrategiesConfig.java

```diff
@@ -96,7 +96,7 @@ public boolean isEnabled() {
      * @return all enabled {@link DiscoveryStrategy} configurations
      */
     public Collection<DiscoveryStrategyConfig> getDiscoveryStrategyConfigs() {
-        return Collections.unmodifiableCollection(discoveryStrategyConfigs);
+        return discoveryStrategyConfigs;
```

Comment:
Can we also rename of this class, `DiscoveryStrategiesConfig` to `DiscoveryStrategyConfigs`. We have such a naming convention in our config collections: replicatedMapConfigs, wanReplicationConfigs, multiMapConfigs